docs/install/install-local-migrate.md

```diff
@@ -298,22 +300,27 @@ You can reuse some of the older Production settings in the `local.env` if you li
 
 ---
 
-## Step 6: Create New Self-Signed Certs for Your Project
-**This step is optional, and does not currently work on Windows.**
+## Step 6: Create new self-signed certs for your project
```

Comment:
Can you bring the caps back, or change all of them? :)  Thanks!

Reply:
@bseeger - I'll add the initial-caps consistency back in my next docs PR. - dkc